src/components/wayfinder/wayfinder.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Modal, { useModalButtonHandler } from "../../core/utils/modal";
+import { useText, withText } from "../../core/utils/text";
+import IconLocation from "../icon-location/icon-location";
+import "./wayfinder.scss";
+import { easyWayUrl } from "../find-on-shelf/getWayfinder";
+import { WayfinderReaponse } from "./wayfinder-types";
+
+const Wayfinder: React.FC<WayfinderReaponse> = ({ viewId, link }) => {
+  const t = useText();
+  const { open } = useModalButtonHandler();
+  let language = document.querySelector("html")?.getAttribute("lang") || "";
+  if (language) {
+    language = `&lang=${language}`;
+  }
+
+  if (viewId === undefined || link === undefined) {
```

Comment:
use non strict compare with null: `variable == null` or `variable != null` in this case you will handle the `null` and `undefined` values.